support-workers/src/main/scala/com/gu/emailservices/DigitalPackEmailFields.scala

```diff
@@ -126,18 +126,22 @@ class DigitalPackEmailFields(
 
   def build(
     paidSubPaymentData: Option[PaymentMethodWithSchedule],
-    readerType: ReaderType
+    readerType: ReaderType,
+    maybeGiftRecipient: Option[GiftRecipient.DigitalSubGiftRecipient]
   ): Either[String, List[EmailFields]] = {
 
     val Purchase = Some
     val Redemption = None
 
     (paidSubPaymentData, readerType) match {
       case (Purchase(paymentInfo), ReaderType.Gift) =>
-        List(
-          giftPurchaserConfirmation(paymentInfo.paymentMethod),
-          giftRecipientNotification
-        ).sequence
+        for {
+          giftRecipient <- maybeGiftRecipient.toRight("Gift redemption must have a gift recipient")
+          emailFieldses <- List(
```

Comment:
`emailFields`?

Reply:
😂

Reply:
oh actually that's what intellij suggests for a double plural! 
![image](https://user-images.githubusercontent.com/7304387/94807930-3a479d80-03e8-11eb-8b6a-3b9113bdd4fe.png)
maybe `emails` would make more sense as fields is a bit of a useless addition in the context but it is important that there are more than one email.